app/src/components/common/Toast.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div
+    v-if="show"
+    class="toast-container"
+    :class="styleType"
+    :style="{ animationDuration: getAnimationDuration }"
+  >
+    <div class="icon">
+      <font-awesome-icon
+        :icon="getIcon"
+      />
+    </div>
+    <slot />
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator'
+
+@Component({})
+export default class Toast extends Vue {
+  @Prop({
+    required: false,
+    default: 'success',
+    validator: (value: string) =>
+        [ 'success', 'danger', 'info', 'warning' ].includes(value)
+  })
+  styleType!: string
+  timeout: number = 0
+  show: boolean = false
+  icons: Record<string, string> = {
+    success: 'fa-check-circle',
+    danger: 'fa-circle-xmark',
+    info: 'fa-info-circle',
+    warning: 'fa-exclamation-triangle',
+  }
+  get getAnimationDuration (): string {
+    return `${this.timeout / 1000}s`;
```

Comment:
```suggestion
    return `${this.timeout / 1000}s`
```

Reply:
Just fixed invalid formatting in recent commit.